dftimewolf/lib/resources.py

```diff
@@ -62,3 +62,9 @@ def GetHelpString(self) -> str:
     short_description = self.contents.get(
         'short_description', 'No description')
     return ' {0:<35s}{1:s}\n'.format(self.name, short_description)
+
+  def GetTestParams(self) -> Optional[list[str]]:
+    """Get the test params from a recipe."""
+    if self.contents.get('test_params', None):
+      return str(self.contents.get('test_params', '')).split(' ')
+    return None
```

Comment:
We could also always return an empty list if no test_params are found (the same truthiness as `None`)

Reply:
Related to the other comment, changed up to recognise the difference between an empty test arg list `[]` and the recipe missing the `test_params` field (now throws exception in the latter case.)